arm-network/2017-06-01/swagger/loadBalancer.json

```diff
@@ -557,10 +558,11 @@
         },
         "protocol": {
           "type": "string",
-          "description": "The transport protocol for the endpoint. Possible values are: 'Udp' or 'Tcp'",
+          "description": "The transport protocol for the endpoint. Possible values are: 'Udp,' 'Tcp,' or 'All.'",
           "enum": [
             "Udp",
-            "Tcp"
+            "Tcp",
+            "All"
```

Comment:
@ravbhatnagar / @johanste Could you please me verify this. Based on my understanding adding new possible values in the existing enum is considered a breaking change.